experimental/golden_response_checker.go

```diff
@@ -209,13 +210,17 @@ func CheckGoldenJSONResponse(t *testing.T, dir string, name string, dr *backend.
 	fpath := path.Join(dir, name+".jsonc")
 
 	expected, err := readGoldenJSONFile(fpath)
-	assert.NoError(t, err)
 	if err != nil {
```

Comment:
@ryantxu this was causing issues for me when new tests were added. it would fail and there was no way to create the file